community/io/src/main/java/org/neo4j/io/ByteUnit.java

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.io;
+
+/**
+ * A ByteUnit is a unit for a quantity of bytes.
+ *
+ * The unit knows how to convert between other units in its class, so you for instance can turn a number of KiBs into
+ * an accurate quantity of bytes. Some loss can occur when quantities are converted between units, due to integer
```

Comment:
since we always returning long values as result of transformations, can you please or point me to a place where we can have rounding error or remove part that describe them